Makefile

```diff
@@ -5,5 +5,6 @@ all:
 
 test:
 	@echo "Running tests..."
+	@if [ -n $(TEST_CONNECTION) ]; then $(MONGODB_BINARIES)/mongosh "$(MONGODB_URI)" --eval "db.runCommand({\"ping\":1})" --&serverSelectionTimeoutMS=10000; fi
```

Comment:
Does this work for TLS clusters? (does MONGODB_URI include the TLS options?)

Reply:
We're running with Auth SSL in some of the tests that have run.

Reply:
I don't think it's working:
```
 [2024/05/14 08:33:06.598] Current Mongosh Log ID:	66438432bc60e6820d5446fe
 [2024/05/14 08:33:06.685] Connecting to:		mongodb://<credentials>@localhost:27017,localhost:27018,localhost:27019/?authSource=admin&replicaSet=repl0&serverSelectionTimeoutMS=2000&appName=mongosh+2.1.1
 [2024/05/14 08:33:08.713] MongoServerSelectionError: connection <monitor> to 127.0.0.1:27017 closed
 [2024/05/14 08:33:08.713] Finished command 'shell.exec' in function 'run tests' (step 2 of 2) in 2.3270065s.
```
https://spruce.mongodb.com/task/drivers_tools_tests_all__os_fully_featured~rhel8_auth~auth_ssl~ssl_test_latest_replica_set_patch_38519e414a412b584f681d01e7f9a2c3831f2468_664383a49c919d0007736bba_24_05_14_15_30_46/logs?execution=0&sortBy=STATUS&sortDir=ASC